Dockerfile

```diff
@@ -1,32 +1,21 @@
 FROM node:4.4
-
-ENV user nodegoat_docker
-ENV workdir /usr/src/app/
-
+ENV WORKDIR /usr/src/app/
+WORKDIR $WORKDIR
+COPY package*.json $WORKDIR
+RUN npm install --production --no-cache
+
+FROM node:4-alpine
```

Comment:
the node image has its own built-in `node` user, perhaps we can use that instead of creating a special USER and GROUP for it?

see reference for the the dockerfile here: https://github.com/nodejs/docker-node/blob/master/Dockerfile-alpine.template

Reply:
I thought about this too but during comparing node user with custom user I assumed that we might need to define our own uid/gid later and of course we need to disable the shell acess for the user. That's why I didn't mention the possibilty of using the existing node user instead of creating another

Reply:
Even if that's the case, I don't think the scope of the project and/or container is to not allow users to login through the shell...

Reply:
Okay then Its better to use the node user, should I wait for confirmtion from other reviewers or should I go and modify it ?

Reply:
Let's wait and see what the other say

Reply:
Yeah, I think your case to allow shell makes sense @lirantal

Reply:
This one is done